python/features/steps/general_purpose_steps.py

```diff
@@ -106,6 +126,31 @@ def check_model_fail(context, model_file):
         raise AssertionError(f"Check cli command succeeded with message: {output_message}")
     context.output_message = output_message
 
+@when('print-defs is called from terminal')
+def print_defs(context):
+    """
+    Run the Print-Defs command
+
+    Args:
+        context: Active context to check against.
+    """
+    exit_code, output_message, std_out = run_cli_command_with_args_with_stdout("print-defs", [])
+    context.exit_code = exit_code
+    context.output_message = output_message
+    context.std_out = std_out
+
+@when('print-defs is called from terminal with the core_only flag')
```

Comment:
Same for here, should be able to genericize this to receive the command and the flag. And that would let you replace some of the other steps currently hard coded.